model/minsky.cc

```diff
@@ -264,6 +265,35 @@ namespace minsky
     GroupPtr g(new Group);
     canvas.setItemFocus(model->addGroup(g));
     m.populateGroup(*g);
+    g->resizeOnContents();	  
+  }
+
+    // Fix for ticket 1080
+  void Minsky::pasteItems()
+  {
+    istringstream is(getClipboard());
+    xml_unpack_t unpacker(is);
+    schema2::Minsky m;
+    xml_unpack(unpacker, "Minsky", m);
+    GroupPtr g(new Group);
+    canvas.setItemFocus(model->addGroup(g));
+    m.populateGroup(*g);
+    if (auto p=g->group.lock())
```

Comment:
p should equal model at this stage.

Reply:
Thanks, that seems much simpler than what I have.